CodeEdit/Features/UtilityArea/TerminalUtility/UtilityAreaTerminalView.swift

```diff
@@ -149,7 +150,7 @@ struct UtilityAreaTerminalView: View {
                                         }
                                     }
                                 )
-                                .frame(height: constrainedHeight - totalFontHeight + 1)
+                                .frame(height: (terminalHeight.isNormal && terminalHeight > 0) ? terminalHeight : 0.0)
```

Comment:
Do you think this would be better here?
```suggestion
                                .frame(height: max(terminalHeight, 0)
```

Reply:
I don't think so. If the value is infinite it'll still throw a runtime warning and max would choose infinity over a real number.